nixos/modules/services/networking/tetrd.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+let
+  tetrd-init = pkgs.writeShellScript "tetrd-init" ''
+    mkdir -p /usr/bin
+    ln -sf ${pkgs.nettools}/bin/{route,ifconfig} /usr/bin
+  '';
```

Comment:
This could be done with `BindReadOnlyPaths=["${pkgs.nettools}/bin/route:/usr/bin/route" "${pkgs.nettools}/bin/ifconfig:/usr/bin/ifconfig" ]` and `UMask="006"`.

Reply:
We tried that already, and there was no way to get it to execute, thus why I tried linking again.

Reply:
It works for me, please double check your setup, this may be due to your `/usr` or `/usr/bin` directories having been created without `UMask="006"`

Reply:
Never mind, it "magically" works now. I'll update.